apps/common/models.py

```diff
@@ -950,3 +951,23 @@ class Meta:
                 fields=["country", "product"], name="common_countryclassified_country_code_product_code_uniq"
             )
         ]
+
+
+class HeaProfile(Model):
+    """
+    A profile to store data associated with a user to be used by the Livelihoods Explorer
```

Comment:
A profile to store data associated with a user to enable a customized user experience.